hq_superset/oauth.py

```diff
@@ -12,6 +12,8 @@ def oauth_user_info(self, provider, response=None):
         if provider == 'commcare':
             logging.debug("Getting user info from {}".format(provider))
             user = self.appbuilder.sm.oauth_remotes[provider].get("api/v0.5/identity/", token=response).json()
+            domains = self.appbuilder.sm.oauth_remotes[provider].get("api/v0.5/user_domains/", token=response).json()
+            session["user_hq_domains"] = domains
```

Comment:
Can store the key in a constant, to re-use.

Reply:
Indeed.